troposphere/static/js/components/admin/ImageRequest.react.js

```diff
@@ -136,141 +136,143 @@ export default React.createClass({
 
         return (
         <div className="request-admin pull-right admin-detail">
-            <div>
-                <h3>Request Information</h3>
+            <div style={{ marginBottom: "20px" }}>
+                <h3 className="t-body-2">Request Information</h3>
                 <div>
-                    Request ID:
-                    {request.get("id")}
+                   Request ID:
+                   {` ${request.get("id")}` }
```

Comment:
Backticks offer the marginal ability to make text more readable by reducing the clutter of the `" + var + "` to more or less just the var. But its easy to throw in arbitrary expressions which completely subvert any gained readability like: `${a.b().somelongfunc()} hello ${var().get} world ${var.props.a}`. 

I think either proposed is fine.

I don't really like backticks for much beside:
```
{`Request ID:  ${requestId}`}
```